src/dataflow/utilities/node.ts

```diff
@@ -1,27 +1,44 @@
-export const NodeTypes = [
+export interface NodeType {
+  name: string;
+  displayName: string;
+}
+
+export const NodeTypes: NodeType[] = [
   {
     name: "Sensor",
+    displayName: "Sensor",
   },
   {
     name: "Number",
+    displayName: "Number",
   },
   {
     name: "Generator",
+    displayName: "Generator",
+  },
+  {
+    name: "Timer",
+    displayName: "Timer (on/off)"
   },
   {
     name: "Math",
+    displayName: "Math",
   },
   {
     name: "Logic",
+    displayName: "Logic",
   },
   {
     name: "Transform",
+    displayName: "Transform",
   },
   {
     name: "Relay",
+    displayName: "Relay",
   },
   {
     name: "Data Storage",
+    displayName: "Data Storage",
```

Comment:
`displayName` is now what appears in the node.  At present the `displayName` is only different for the timer, but we introduced a bug a month or so ago when we changed the case of the first letter on the node `name` field (since the `name` was both user facing and used internally to check the node type).  Future requests to change the node displayed name should be much safer.